test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -298,9 +301,9 @@ instance MonadSlotsData (TestInitMode ssc) where
     putSlottingData = putSlottingDataDefault
 
 instance MonadSlots (TestInitMode ssc) where
-    getCurrentSlot = getCurrentSlotSimple
-    getCurrentSlotBlocking = getCurrentSlotBlockingSimple
-    getCurrentSlotInaccurate = getCurrentSlotInaccurateSimple
+    getCurrentSlot = getCurrentSlotSimple =<< mkSimpleSlottingVar
+    getCurrentSlotBlocking = getCurrentSlotBlockingSimple =<< mkSimpleSlottingVar
+    getCurrentSlotInaccurate = getCurrentSlotInaccurateSimple =<< mkSimpleSlottingVar
     currentTimeSlotting = currentTimeSlottingSimple
```

Comment:
Is it ok that `currentTimeSlotting` is potentially not coherent with `getCurrentSlotXX`?